### ISSUE TITLE
Framework color lerping logic assumes it can always use Color.lerp

### ISSUE THREAD

**Issue by dnfield** [flutter/flutter#48684]
For instance, ColorTween takes a Color, which could be an HSVColor or HSLColor, which, have their own lerping algorithms and may be not precisely compatible with `Color.lerp`.

See also https://github.com/flutter/flutter/issues/48534

**Comment by pedromassangocode** [comment]
Is this still a valid issue? I'm not sure how to verify it in the framework.

**Comment by dnfield** [comment]
Yes - the basic problem right now is that `lerp` is a static method on the various `Color` subclasses, and instead should probably be an instance method like `lerpTo` or `lerpFrom`.
